### PR TITLE
maked change in workdetails page

### PR DESCRIPTION
**Pull request by shahzad-chauddhary**
change in workdetail

### CODE REVIEW

**Comment by shahzad-mycroft** [comment]
add  .idea/* in gitignore
add *pyc in gitignore

delete .idea from this PR
delete all .pyc files from this PR

**Comment by shahzad-chauddhary** [comment]
close pR

File: app/models.py [hunk 1]
```diff
@@ -3,3 +3,13 @@
 
 class MobileApp(models.Model):
     img = models.ImageField(upload_to='gallery')
+
+class WorkDetail(models.Model):
+    clientnam = models.CharField(max_length=15)
```
Comment (shahzad-mycroft) [inline app/models.py:8]: it be client_name

Always keep the small letter with an underscore.
Always keep meaningful names.

File: app/models.py [hunk 2]
```diff
@@ -3,3 +3,13 @@
...
+    category = models.CharField(max_length=15)
+    proj_date = models.DateField()
```
Comment (shahzad-mycroft) [inline app/models.py:10]: Same as above project_date

File: app/models.py [hunk 3]
```diff
@@ -3,3 +3,13 @@
...
+    cimg = models.ImageField(upload_to='details')
+    des1 = models.CharField(max_length=150)
+    des2 = models.CharField(max_length=150)
```
Comment (shahzad-mycroft) [inline app/models.py:13]: same as above

File: app/models.py [hunk 4]
```diff
@@ -3,3 +3,13 @@
...
+    thumb_img1 = models.ImageField(upload_to='details')
+    thumb_img2 = models.ImageField(upload_to='details')
```
Comment (shahzad-mycroft) [inline app/models.py:15]: same as above

Comment (shahzad-chauddhary) [reply app/models.py:15]: ok

**Pull request merged**